app/controllers/errors_controller.rb

```diff
@@ -4,7 +4,7 @@ class ErrorsController < ApplicationController
     define_method error_code.to_sym do
```

Comment:
Can you also remove 502 from the list above? We don't have a template for it in app/views/errors, or a route for it in config/routes.rb. It's a static error page that doesn't need to be processed by this controller.

Reply:
Thanks, this will be done in the next commit (and included in the commit message).

Reply:
Since we're responding to all formats in the same way, you should be able to write:
```rb
%w(403 404 422 500).each do |error_code|
  define_method error_code.to_sym do
    render error_code, status: error_code.to_i, formats: :html
  end
end
```
You can also leave out the content type, since `formats: :html` is already specified.

When you're making changes to pull requests, we generally prefer that you don't force push. That way it's easier for reviewers to view changes compared to the previous round of review.